packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -678,27 +677,18 @@ export class AutolinkWindows {
   }
 
   public async ensureXAMLDialect() {
-    let changesNeeded = false;
```

Comment:
@NickGerleman do we need to revert this?
Will Picker (and potentially other consuming projects) require this mechanism if they keep using packages.confg?